hutch_python/happi.py

```diff
@@ -73,21 +87,22 @@ def get_happi_objs(
         results = client.search(name=name)
         containers.extend(res.item for res in results)
 
-    # also any device with the same beamline name
-    # since lightpath only grabs lightpath-active devices
-    beamlines = {it.beamline for it in containers}
-    beamlines.add(endstation.upper())
-
-    for line in beamlines:
-        # Assume we want hutch items that are active
-        # items can be lightpath-inactive
-        reqs = dict(beamline=line, active=True)
-        results = client.search(**reqs)
-        blc = [res.item for res in results
-               if res.item.name not in dev_names]
-        # Add the beamline containers to the complete list
-        if blc:
-            containers.extend(blc)
+    if load_level >= DeviceLoadLevel.STANDARD:
+        # also any device with the same beamline name
+        # since lightpath only grabs lightpath-active devices
```

Comment:
I missed this justification previously, I wonder if this would cause any issues for non-lightpath devices in the hutches if we move to UPSTREAM mode. Do we need to have any sort of special handling for these kinds of devices?

Reply:
It's a good point.  My inclination is that if hutches want those devices they should just pick `STANDARD`, since I don't know how to find / filter a level between `STANDARD` and `UPSTREAM`

Something like this might be nice, but inaccessible given our current metadata
`UPSTREAM`
`UPSTREAM + lightpath-inactive-but-on-beampath`
`STANDARD`

Reply:
That's a good counterpoint- how would we even know which devices to pick? Needs more thought.